### PR TITLE
Attachment automatically saved when done.

### PR DESCRIPTION
**Pull request by CaiqueMitsuoka**
This PR:
* Create two functions callbacks for 2 events: Click button save and done attachment upload.
* Removes the visual feedback to save the story, since it isn't necessary

### CODE REVIEW

File: app/assets/javascripts/views/story_view.js [hunk 1]
```diff
@@ -278,7 +279,7 @@ module.exports = FormView.extend({
     }
   },
 
-  saveEdit: function(event) {
+  saveEdit: function(event, shoudHideForm) {
```
Comment (castanhob) [inline app/assets/javascripts/views/story_view.js]: Typo: `shoudHideForm` deveria ser `shouldHideForm`?

File: app/assets/javascripts/views/story_view.js [hunk 2]
```diff
@@ -292,8 +293,10 @@ module.exports = FormView.extend({
 
     this.model.save(null, { documents: documents,
       success: function(model, response) {
-        that.model.set({editing: false});
         that.enableForm();
+        if(shoudHideForm){
```
Comment (castanhob) [inline app/assets/javascripts/views/story_view.js]: Typo: `shoudHideForm` deveria ser `shouldHideForm`?

File: app/assets/javascripts/views/story_view.js (hunk 1 above)
Comment (CaiqueMitsuoka) [inline app/assets/javascripts/views/story_view.js]: should

File: app/assets/javascripts/views/story_view.js [hunk 3]
```diff
@@ -698,5 +700,15 @@ module.exports = FormView.extend({
       $div.append(content.control);
     }
     return div;
+  },
+
+  attachmentDone: function(event){
+    if (!this.model.isNew()) {
+      this.saveEdit(event,false);
```
Comment (vitortalaia) [inline app/assets/javascripts/views/story_view.js]: What about a space after the comma here? 💅

File: app/assets/javascripts/views/story_view.js [hunk 4]
```diff
@@ -698,5 +700,15 @@ module.exports = FormView.extend({
       $div.append(content.control);
     }
     return div;
+  },
+
+  attachmentDone: function(event){
+    if (!this.model.isNew()) {
+      this.saveEdit(event,true);
```
Comment (daniloisr) [inline app/assets/javascripts/views/story_view.js]: missing a space here too

**Pull request merged**